packages/ember-htmlbars/tests/helpers/bind_attr_test.js

```diff
@@ -599,3 +599,25 @@ QUnit.test("src attribute bound to null is not present", function() {
 
   ok(!view.element.hasAttribute('src'), "src attribute not present");
 });
+
+QUnit.test('specifying `<div {{bind-attr style=userValue}}></div>` is [DEPRECATED]', function() {
+  view = EmberView.create({
+    userValue: '42',
+    template: compile('<div {{bind-attr style=view.userValue}}></div>')
+  });
+
+  expectDeprecation(function() {
+    runAppend(view);
+  }, /Allowing dynamic content in a `style` attribute is deprected, please preform a security audit and once verified use `{{{property}}}` instead of `{{property}}`./);
```

Comment:
The key point here is that binding a dynamic style is inherently unsafe, therefore users should closely review/audit to see if that dynamic style value is exposed to user input in any way. So we are deprecating the concept of binding to style with an escaped value. 

I will update this deprecation message to make it clear that they should refactor {{bind-attr style=foo}} into the new syntax (forcing folks to think about the security implications).

When we deprecate bind-attr (I plan on making that in an early 1.12 beta), we will tell folks to use HTMLBars attribute syntax for everything.